### PR TITLE
Fix LegacyInitOperation `toJson`

### PR DESCRIPTION
**Pull request by truongsinh**
`toJson()` should always return a `<String, dynamic>` not something else, especially not `String` (`InitOperation` is working fine, only `LegacyInitOperation` is broken)

Fix https://github.com/zino-app/graphql-flutter/issues/134

After merging to master, we should pump patch version so that people can use this fix.

### CODE REVIEW

File: packages/graphql/lib/src/websocket/messages.dart [hunk 1]
```diff
@@ -74,15 +74,15 @@ class LegacyInitOperation extends InitOperation {
...
+  Map<String, dynamic> toJson() {
     final Map<String, dynamic> jsonMap = <String, dynamic>{};
     jsonMap['type'] = type;
 
     if (payload != null) {
       jsonMap['payload'] = json.encode(payload);
     }
 
-    return json.encode(jsonMap);
+    return jsonMap;
```
Comment (mainawycliffe) [inline packages/graphql/lib/src/websocket/messages.dart]: Why are we removing JSON encoder?

Comment (truongsinh) [reply packages/graphql/lib/src/websocket/messages.dart]: as explain the description of this PR, toJson() should always return a <String, dynamic> not something else, especially not String.

**Pull request merged**